virt/install/preparing-cluster-for-virt.adoc

```diff
@@ -67,6 +67,16 @@ Live migration has the following requirements:
 * Sufficient RAM and network bandwidth.
 * If the virtual machine uses a host model CPU, the nodes must support the virtual machine's host model CPU.
 
+[NOTE]
+====
+You must ensure that there is enough memory in the cluster to support concurrent live migrations. The default xref:../../virt/live_migration/virt-live-migration-limits.adoc#virt-live-migration-limits[number of migrations running in parallel] in the cluster is 5. You can calculate the required spare memory by using the following equation:
+
+----
+Required spare memory in the cluster = number of VMs * number of nodes that are allowed to drain in parallel
```

Comment:
`Required spare memory in the cluster = number of VMs * number of nodes that are allowed to drain in parallel`

should be

`Required spare memory in the cluster = sum of memory of number of nodes that are allowed to drain in parallel`

Reply:
@jeniferh please keep me honest

Reply:
Hmm actually thinking on it more, slightly more complex: 
Required spare memory in the cluster = total VM memory requests that will migrate. This can be calculated by considering the max number of nodes draining at once (depending on MCP config), and looking at the highest total VM memory request allocations across nodes. When draining a whole cluster, VMs will need that spare memory request capacity to schedule a migration pod and fully drain the top allocated nodes (total nodes draining in parallel x top mem requests).